doc/related_projects.rst

```diff
@@ -259,10 +262,8 @@ Other packages useful for data analysis and machine learning.
 Recommendation Engine packages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
- - `GraphLab
-   <https://turi.com/products/create/docs/graphlab.toolkits.recommender.html>`_
-   Implementation of classical recommendation techniques (in C++, with
-   Python bindings).
+- `GraphLab <https://turi.com/products/create/docs/graphlab.toolkits.recommender.html>`_ Implementation
```

Comment:
Please revert this, since it is not related to the title of this PR.

Reply:
Have reverted it, but the alignment looks off for the Graphlab bullet in the "Recommendation Engine packages" section, should we raise a new issue for it.